src/crime_vancouver_eda.md

```diff
@@ -0,0 +1,55 @@
+# Exploratory data analysis of Crimes in Vancouver data set
+
+# Summary of the data set
+
+The data set used in this project comes from the Vancouver Police Department Open Data (The Vancouver Police Department, 2021) and can be found here (https://geodash.vpd.ca/opendata/). Each row in the data set represents a reported crime in Vancouver, including crime type, date (in year, month, day, hour, minute), block address, and neighbourhood name. The data set covers cases from 2003 to 2021 of a total of 782,749 cases. For the visualization purpose, we focus on the past ten years' data here (from 2020 to 2021).
+Below we show the number of reported crimes for each neighbourhoods. Note there are 24 neighbourhoods in Vancouver.
+
+![](figure-gfm/neighbour_crimes.png)<!-- -->
```

Comment:
Can we in general make the tables in centred and add descriptions?
As mentioned at Tiff's class we better use <img src="xxx"> for embedding images, such that you can control the width and height of the image.

Reply:
I can try to reformat the images to the center.

Can you collaborate on what you mean by 'adding descriptions'? The table aims to support the paragraphs and it has table name. What kind of description you want me to add here? @thomassiu

Reply:
@sy25wang  something like below, the alt part. But this is nice to have only.

\<img src="pic_trulli.jpg" alt="Italian Trulli">